src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.datasource.url=jdbc:h2:mem:todolist
+spring.datasource.username=admin
+spring.jpa.defer-datasource-initialization=true
```

Comment:
jeżeli korrzystasz z automatycznego generowania schematu, to zamiast dodawać tę własność, lepiej jest użyć pliku import.sql zamiast data.sql